dev/test-studio/assetSources/imageAssetSource.tsx

```diff
@@ -1,10 +1,10 @@
 import {ImagesIcon} from '@sanity/icons'
```

Comment:
Rverted the changes on this and the following component as they are test-studio components, shouldn't be consuming from sanity-ui